src/tt_citites_skylines/exporter_dae.rb

```diff
@@ -0,0 +1,109 @@
+#-------------------------------------------------------------------------------
+#
+# Thomas Thomassen
+# thomas[at]thomthom[dot]net
+#
+#-------------------------------------------------------------------------------
+
+module TT::Plugins::CitiesSkylinesTools
+
+  def self.export_asset_dae
+    # Find the destination for the asset.
+    default_path = self.find_asset_path
+    filename = self.get_model_dae_name
+    if Sketchup.platform == :platform_win
+      dae_filter = "COLLADA File (*.dae)|*.dae||"
```

Comment:
Can probably combine some code between the two - let the savedialog present both formats and then choose the exporter based on that. I can help out on that. But this would be good to do in a separate branch.